File: fundamentos/desafio-js/src/questions/how-many-acc-at-ag-47.ts
import { IAccount } from "../i-account";

/**
 * Quantos clientes estão na agência 47?
 */
export default function howManyAccAtAg47(...accounts: Array<IAccount>) {
  const sum = accounts
    .filter(acc => acc.agencia == 47)
    .length;

  console.info(`Quantos clientes estão na agência 47? ${sum}`);
}

File: fundamentos/desafio-dart/lib/questions/how_many_acc_at_ag_47.dart
import 'package:desafio_dart/domains/account.dart';

/// Quantos clientes estão na agência 47?
void howManyAccAtAg47(List<Account> accounts) {
  final sum = accounts.where((acc) => acc.agencia == 47).length;

  print('Quantos clientes estão na agência 47? ${sum}');
}
